src/test/setupTests.js

```diff
@@ -0,0 +1,8 @@
+// import { configure } from 'enzyme';
+// import Adapter from 'enzyme-adapter-react-16';
+
+//TODO: switch to import
+let Adapter = require('enzyme-adapter-react-16');
```

Comment:
Better to use `const` for immutability: 
```suggestion
const Adapter = require('enzyme-adapter-react-16');
```